spec/hardware/virtualization.fmf

```diff
@@ -0,0 +1,14 @@
+summary:
+    Features related to virtualization
+example:
+  - |
+    # Require a guest which supports virtualization
+    virtualization:
+        is-supported: false
+  - |
+    # Ask for a virtualized guest using kvm hypervisor
+    virtualization:
+        is-virtualized: true
+        hypervisor: kvm
```

Comment:
Maybe we could mention at least some hypervisors, it's a field similar to `arch` - some are common, some different people give different names, so far I've met the following: `xen`, `nitro`, `kvm`, `powerkvm`, `powervm`, `hyperv`, `vmware`.

Reply:
It probably doesn't make sense to make it an enum though, just in case. (I just noticed Artemis does treat this field as one, I'll fix that).

Reply:
Is there a real difference between `kvm` and `powerkvm`? By the way, uses names like `KVM`, `PowerKVM`, `PowerVM`, but it seems to perform case-insensitive match when evaluating host requirements, so passing it a value like `kvm` or `powervm` should work as well.

Reply:
I would expect there *might* be some differences, given that PowerKVM is built upon KVM, by IBM, therefore they may add extra features. It should be compatible, whatever that might mean in terms of KVM, but still, it does look like a different hypervisor to me, IIUIC.

> PowerKVM is open source server virtualization that is based on the IBM POWER8® processor technology. It includes the Linux open source technology of KVM virtualization, and it complements the performance, scalability, and security qualities of Linux.

Reply:
Do you know whether KVM on Power machines is always PowerKVM? I am asking because I saw both PowerKVM and KVM ppc64le machines in Beaker and I don't know if it means that they are using two slightly different virtualization technologies or if the naming is just inconsistent.

Reply:
I don't know the answer, Beaker folks would be a much better source, I for one would assume two different hypervisors.

Reply:
> It probably doesn't make sense to make it an enum though, just in case.

Agreed. Let's keep it rather open. Examples added in e03ff940.

Reply:
Plus fixed a `true` / `false` typo.